frontend/pages/software/ManageSoftwarePage/ManageSoftwarePage.tsx

```diff
@@ -181,12 +280,13 @@ const ManageSoftwarePage = ({
     state: ITeamsDropdownState
   ): JSX.Element | null => {
     if (
-      (state.isGlobalAdmin || state.isGlobalMaintainer) &&
-      state.teamId === 0
+      canAddOrRemoveSoftwareWebhook &&
+      (!isPremiumTier || state.teamId === 0) &&
+      !isLoadingSoftwareVulnerabilitiesWebhook
```

Comment:
This line ensures we don't run into detachment issues on e2e test:
     CypressError: Timed out retrying after 20050ms: `cy.click()` failed because this element is detached from the DOM.